### PR TITLE
improve descriptions of settings

### PR DESCRIPTION
**Pull request by keikoro**
I originally only wanted to improve the info on GitHub/GitLab tokens, then saw the file could use further refactoring.

I split my changes into several commits so you can decide which ones you want to keep.

In the last commit, I reformatted the descriptions for the last two settings to match the rest of the file; I would have liked to reword the one for CSS so it makes more sense, but didn't really understand how it's supposed to be used.

### CODE REVIEW

**Comment by facelessuser** [comment]
Thanks for the pull! Yeah, there seem to be a ton of typos and such in there, so this is much appreciated. I'll take a look and let you know if I have any additional comments.

File: MarkdownPreview.sublime-settings [hunk 1]
```diff
@@ -214,36 +234,36 @@
...
 
-        true - Any file with matching a .markdown_filetype extension with .css will be loaded as an override
+        true - Any file with matching a .markdown_filetype extension with .css will be loaded as an override.
         false - Matching files ignored
     */
     "allow_css_overrides": true,
 
     /*
-        Specify a HTML template file to render your markdown within.
+        Specify an HTML template file within which to render your Markdown.
```
Comment (facelessuser) [inline MarkdownPreview.sublime-settings:245]: This is probably more correctly specified with `with which` opposed to `within which`.

**Comment by facelessuser** [comment]
Overall looks good, I didn't have too many comments. If you can address the single comment, we can go ahead and merge this in.

**Comment by keikoro** [comment]
I modified the comment/commit in question as per your suggestion.

**Comment by facelessuser** [comment]
@gir-bot lgtm

**Pull request merged**